src/main/resources/application.yaml

```
@@ -187,6 +187,9 @@ darts:
   dar-pc-notification:
     url-format: http://{0}/VIQDARNotifyEvent/DARNotifyEvent.asmx
   events:
+    duplicates:
+      earliest-removable-event-date: ${MODERNISED_DARTS_START_DATE:2024-07-16}
+      clear-up-window: 2
```

Is this specified in days?

Think it might be worth adding a env var here so that it can be overridden in flux if needed.

```suggestion
      clear-up-window: ${DUPLICATE_EVENTS_CLEAR_UP_WINDOW_DAYS:2}
```